File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int h = -1;
    while (h < 0 || h > 23)
    {
        printf("Height: ");
        h = get_int();
    }
    for (int i=0; i < h; i++)
    {
        for (int j=0; j < h-i-1; j++)
        {
            printf(" ");
        }
        for (int j=0; j < i+1; j++)
        {
            printf("#");
        }
        printf("  ");
        for (int j=0; j < i+1; j++)
        {
            printf("#");
        }
        printf("\n");
    }
}

File: pset6/mario.py
import cs50

h = -1
while (h < 0 or h > 23):
    print("Height: ", end = "")
    h = cs50.get_int()
    
for i in range(h):
    line = ""
    for j in range(h-i-1):
        line += " "
    for j in range(i+1):
        line += "#"
    line += "  "
    for j in range(i+1):
        line += "#"
    print(line)
